tests/drivers/test_oai_xml.py

```diff
@@ -67,3 +67,9 @@ def test_marc21(mock_marc21):
     assert len(df) == 182, "expected number of rows"
     assert len(df.columns) == 92, "expected number of columns"
     assert "245_a" in df.columns, "marc field 245 subfield a extracted"
+
+
+def test_wait(mock_oai_dc):
+    # ensure that the wait option can be used
+    oai = OaiXmlSource("https://example.org", "oai_dc", wait=2)
```

Comment:
This is kind of a lame test since it doesn't actually test that the sleep is happening, but it does ensure that a wait value can be passed in and that the iterator class is being created ok.